hier.c

```diff
@@ -250,4 +251,6 @@ void hier()
   c(auto_qmail,"man/cat8","qmail-smtpd.0",auto_uido,auto_gidq,0644);
   c(auto_qmail,"man/man8","qmail-command.8",auto_uido,auto_gidq,0644);
   c(auto_qmail,"man/cat8","qmail-command.0",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"man/cat8","spawn-filter.8",auto_uido,auto_gidq,0644);
```

Comment:
The .8 file should go to man8, not cat8.

Reply:
Fixed